processor/src/main/java/com/linecorp/decaton/processor/ProcessingContext.java

```diff
@@ -60,9 +66,53 @@
      * Once this method called within {@link DecatonProcessor#process} method, caller *MUST* call
      * {@link DeferredCompletion#complete()} or {@link ProcessingContext#retry()} method in any cases.
      * Otherwise consumption will stuck in short future and no new task will be given to the processor.
-     * @return a {@link DeferredCompletion} which can be used to tell the result of processing asynchronously.
+     * @return a {@link Completion} which can be used to tell the completion of processing asynchronously.
+     */
+    default Completion deferCompletion() {
+        return deferCompletion(() -> false);
+    }
+
+    /**
+     * Tells the completion of this processing should be postponed and processor can accept next task.
+     * Once this method called within {@link DecatonProcessor#process} method, caller *MUST* call
+     * {@link DeferredCompletion#complete()} or {@link ProcessingContext#retry()} method in any cases.
+     * Otherwise consumption will stuck in short future and no new task will be given to the processor.
```

Comment:
ok, let me add following sentence.

```
     * To avoid that with some risks, consider setting
     * {@link ProcessorProperties#CONFIG_DEFERRED_COMPLETE_TIMEOUT_MS}.
```